schemas/vax.schema

```diff
@@ -0,0 +1,35 @@
+SET client_min_messages TO WARNING;
+BEGIN;
+
+DROP SCHEMA IF EXISTS $PG_SCHEMA_VAX CASCADE;
+CREATE SCHEMA $PG_SCHEMA_VAX;
+
+CREATE TABLE $PG_SCHEMA_VAX.age (
+id   integer PRIMARY KEY,
+name varchar(28) NOT NULL
+);
+
+CREATE TABLE $PG_SCHEMA_VAX.race (
+id   integer PRIMARY KEY,
+name varchar(37) NOT NULL
+);
+
+CREATE TABLE $PG_SCHEMA_VAX.vax (  -- hypertables cannot be unlogged
+locale         VARCHAR(20) NOT NULL, 
```

Comment:
I think this is a good idea, I'll implement it. There is some region level data that would not map to a locale (e.g. `Region 9`). I assume those correspond to CDC specific regions. Any thoughts on how to handle those?

Reply:
Oh yes, I see that now. Looks like they are groups of states. There's probably no disease dynamics for those regions so maybe we could just drop them.

Reply:
Cool, I've just done an inner join on the tables so we are left with 50 states + DC. Now the `vax` table has columns:

* `locale_name`: the state name (this could be renamed to `admin2` if we'd like)
* `locale_id`: the `id` from the corresponding record in `main.locale`

Reply:
That sounds good. We could keep `locale_name` for error checking or remove it not to duplicate info from `main.locale`.

Reply:
Any preference on this? 

Otherwise I think all the outstanding issues are resolved. I do feel like adding some tests to spot check all the transforms are correct (I've done this manually) would be beneficial.

Reply:
I'd lean towards just linking to `main.locale`.

I agree about tests. Should I merge now or were you thinking about adding some tests before?

Reply:
Sounds good, I'll make that schema change. 

I've almost finished the test but did it through a new CLI argument so I think it makes more sense to have a separate PR for that.

Reply:
Thanks, Brandon.  Sounds good about the test.

Reply:
Done!